static/app/views/organizationGroupDetails/grouping/grouping.tsx

```diff
@@ -118,12 +119,12 @@ function Grouping({api, groupId, location, project, organization}: Props) {
     }
   }
 
-  function setCurrentGrouping() {
-    const currentGrouping = groupingLevels.find(groupingLevel => groupingLevel.isCurrent);
-    if (!currentGrouping) {
+  function setSecondGrouping() {
+    const secondGrouping = groupingLevels[1];
```

Comment:
We need to basically revert this once the Regroup butotn is available. Otherwise it's way too confusing. Or we go through more design sessions

Reply:
I think we can discuss it a bit more in the next design session